src/driver/driver_api.cc

```diff
@@ -128,63 +154,219 @@ transform::Pass Filter(FCond fcond) {
   return tir::transform::CreatePrimFuncPass(fpass, 0, "Filter", {});
 }
 
-IRModule lower(te::Schedule sch, const Array<te::Tensor>& args, const std::string& name,
-               const std::unordered_map<te::Tensor, tir::Buffer>& binds) {
-  Array<ObjectRef> out_arg_list;
+Array<tvm::transform::Pass> CreatePassList(bool enable_loop_partition, bool for_te_schedule) {
```

Comment:
Is there a better way to handle the creation of the pass list? Should I split this function into two, one for the te schedules and one for the IRModule and primfuncs?